lmdeploy/lite/utils/calib_dataloader.py

```diff
@@ -276,7 +282,12 @@ def get_pileval(tokenizer, nsamples, seed, seqlen=512):
     ], None
 
 
-def get_calib_loaders(name, tokenizer, nsamples=128, seed=0, seqlen=2048):
+def get_calib_loaders(name,
+                      tokenizer,
+                      nsamples=128,
+                      seed=0,
+                      seqlen=2048,
+                      model=''):
```

Comment:
model is not used

Reply:
My bad, there seem to be some issues when rebasing.